src/KubernetesClient/KubernetesClientConfiguration.ConfigFile.cs

```diff
@@ -411,7 +407,6 @@ public static string RenewAzureToken(string tenantId, string clientId, string ap
             throw new KubeConfigException("Refresh not supported.");
         }
 
-#if NETSTANDARD2_0
```

Comment:
Not sure what impact this will have in terms for backwards compatibility.
Will this cause any harm if this block of code runs for non .Net core clients?

Reply:
I'll try running some more testing on this in a bit. If others could test different setups as well, that would probably be the most ideal.